borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,86 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
+                type: integer
+                description: |
+                    The ID of the Zabbix item used for collecting data.
+                    Unique across the entire Zabbix system.
+                example: 55105
+            host:
+                type: string
+                description: |
+                    Host name where the item is stored. Required if "itemid" is not set.
```

Comment:
FYI this description and the next one each needed to be wrapped to 80 lines to make tests pass.